build.psake.ps1

```diff
@@ -243,7 +243,11 @@ Task Test -depends Build {
     Import-Module Pester
     try {
         Microsoft.PowerShell.Management\Push-Location -LiteralPath $TestRootDir
-        Invoke-Pester
+        $TestResult = Invoke-Pester -PassThru -Verbose:$VerbosePreference
```

Comment:
Have you tried this from VSCode?  When there are Pester test failures, we have a VSCode problem matcher that deciphers the Pester output and populates the VSCode Problems panel.  This looks like VSCode will no longer see the output.  It's all captured in $TestResult.